FakeXrmEasy.Tests.Shared/FakeContextTests/QualifyLeadTests/QualifyLeadTests.cs

```diff
@@ -139,5 +139,36 @@ public void Check_if_Account_was_associated_with_Opportunity()
 
             Assert.NotNull(opportunity.CustomerId);
         }
+
+        [Fact]
+        public void Status_of_qualified_Lead_should_be_qualified()
+        {
+            var context = new XrmFakedContext();
+            context.ProxyTypesAssembly = Assembly.GetExecutingAssembly();
+            var service = context.GetFakedOrganizationService();
```

Comment:
Hi @BetimBeja , this is amazing. Just a few minor change, GetFakedOrganisationService is deprecated, we should use only GetOrganisationService from now on.

```suggestion
            var service = context.GetOrganizationService();
```

Reply:
I copied another test case and modified it for my specific test and didn't think about this. I guess i should have not been lazy...
Also we should refactor existing tests to not use GetFakedOrganizationService.

Reply:
@jordimontana82 , i think you can merge the pull request now 👍